nodejs/README.md

```diff
@@ -0,0 +1,58 @@
+---
+display_name: nodejs
+description: Install Node.js via nvm
+icon: ../.icons/node.svg
+maintainer_github: TheZoker
+verified: true
```

Comment:
@mafredri @bpmct What is your opinion on accepting modules as verified?
There is a previous request in #92

Reply:
Ah I just copied that from another module, I can also remove it if requested

Reply:
We originally published those modules and plan to maintain them so they are verified. We have yet to devise a criteria to mark community contributions as verified.

Reply:
I see. So I would suggest to remove this line for now?

Reply:
Yes, setting it ad `verified: false` until we decide on a criteria makes sense. We can change statues in a follow up PR.